crates/egui/src/widgets/label.rs

```diff
@@ -9,10 +9,11 @@ use self::text_selection::LabelSelectionState;
 /// Usually it is more convenient to use [`Ui::label`].
 ///
 /// ```
+/// # use egui::text::TextWrapMode;
 /// # egui::__run_test_ui(|ui| {
 /// ui.label("Equivalent");
 /// ui.add(egui::Label::new("Equivalent"));
-/// ui.add(egui::Label::new("With Options").wrap(false));
+/// ui.add(egui::Label::new("With Options").wrap_mode(TextWrapMode::Truncate));
```

Comment:
This becomes quite verbose.

I think we should consider having two helpers:

* `.truncate() == .wrap_mode(TextWrapMode::Truncate)`
* `.wrap() == .wrap_mode(TextWrapMode::Wrap)`

They then no longer have any `bool` arguments, which could potentially make migration a bit more confusing for users, but the end API is pretty nice:

`ui.add(egui::Label::new("Foo").wrap())`

Reply:
Totally agree with that—was reluctant because of the intrusive breaking change, but the end result is worth it!